eternalcore-core/src/test/java/com/eternalcode/core/util/TestScheduler.java

```diff
@@ -0,0 +1,92 @@
+package com.eternalcode.core.util;
+
+import com.eternalcode.commons.scheduler.Scheduler;
+import com.eternalcode.commons.scheduler.Task;
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
+
+public class TestScheduler implements Scheduler {
```

Comment:
okey we removed "user controller tests" and other useless tests - that is okay. But why you added "TestScheduler" and "IntegrationTestSpec" ?